### ISSUE TITLE
[high] : Many pages: Chrome : The 'More' button is not accessible using screen reader.

### ISSUE THREAD

**Issue by Stutikuls** [Expensify/App#8527]
## Action Performed:
1. Using Chrome, open URL staging.new.expensify.com
2. Navigate to Setting using Tab key and press Enter to select it.
3. Navigate to any workspace using Tab key and press using Enter to select it.
4. Navigate to More button using Tab key and press Enter to activate it.
5. Observe the behavior.

## Expected Result:
All the interactive elements must be operable using screen reader. User should be able to accessible 'More' button by using Enter key.

## Actual Result:
The 'More' button is not accessible using screen reader. When pressing the more button using screen reader different content appeared.

**Other Occurrences:**
>Same issue also repro on PR #8581, #8906
>PR #8730, #9220, #9193 -Call icon is not accessible using screen reader, 
>PR #7849, #8739 -Download button is not accessible using screen reader.
>PR #9055 -Profile picture is not accessible using screen reader.

## Workaround:
Yes

## Area issue was found in:
Settings -> More

## Failed WCAG checkpoints
4.1.1

## User impact:
The Screen reader users are not able to use the 'More' button functionality as mouse/keyboard users.

## Suggested resolution:
Make sure the more button performs the same action for both keyboard and screen reader users, and be sure the screen reader announces the correct name of the control.

Refer to:
https://www.w3.org/TR/wai-aria-practices-1.1/examples/button/button.html

## Platform:

- Web

**Version Number:** v1.1.49-1
**Reproducible in staging?:** Yes
**Reproducible in production?:** Yes
**Issue reported by:** Stuti

https://user-images.githubusercontent.com/97600950/162178209-edcfb316-46da-463d-b7e8-dae73b487acb.mp4
